packages/components/src/VirtualizedList/CellActions/CellActions.component.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import { Actions } from '../../Actions';
+
+import theme from './CellActions.scss';
+
+/**
+ * Cell that renders actions
+ */
+function CellActions({ cellData }) {
+	return (
+		<div className={`tc-list-actions ${theme['tc-list-actions']}`}>
+			<Actions
+				actions={cellData}
+				hideLabel
+				link
+			/>
+		</div>
+	);
+}
+
+CellActions.displayName = 'VirtualizedList(CellActions)';
+CellActions.propTypes = {
+	/** The cell value */
```

Comment:
Please remove unnecessary comment. Or at least use line comments instead of block ones.